pypeit/par/pypeitpar.py

```diff
@@ -4583,6 +4586,16 @@ def __init__(self, tolerance=None, dry_run=None, match_using=None, exclude_slit_
         descr['dry_run'] = "If set, the script will display the matching File and Object Ids " \
                            "but will not flux, coadd or archive."
 
+        # Forces coadding using counts instead of flux
+        defaults['ignore_flux'] = False
+        dtypes['ignore_flux'] = bool
+        descr['ignore_flux'] = "If set, the script will only coadd non-fluxed spectra even if flux data is present. Otherwise fluxed spectra are coadded if all spec1ds have been fluxed calibrated."
+
+        # Enables a flux calibration after coadding
+        defaults['flux'] = False
+        dtypes['flux'] = bool
+        descr['flux'] = "If set, the script will flux calibrate using archived sensfuncs before coadding."
+
```

Comment:
Sorry this is confusing. I'll try to update the docs to explain this better. Essentially `flux` tells pypeit_collate_1d to run flux calibration, overwriting any previous `OPT_FLAM` `BOX_FLAM` information in the SpecObj. This always happens before coadding.  

`ignore_flux` causes pypeit_collate_1d to ignore flux information for coadding. If it's false, pypeit_collate_1d will coadd the fluxed data if available, otherwise it will use counts. If it's true it will always use counts. It's intended to preserve the behavior of pypeit_collate_1d before this PR, which never used fluxed data.